client_internals_it_test.go

```diff
@@ -238,14 +238,6 @@ func TestClientInternal_OrderedMembers(t *testing.T) {
 	it.Eventually(t, func() bool {
 		return sameMembers(targetUUIDs, ci.OrderedMembers())
 	})
-	types.NewUUID()
```

Comment:
Is this related to the PR?

Reply:
No, this is an accidental leftover from an earlier PR.